lib/entity.js

```diff
@@ -10,16 +9,39 @@ function Entity (entity) {
   this.type = entity.type || 'Unknown';
   this.name = entity.name || this.id;
 
-  this.experience = 0;
+  this.EXP = 0;
+  this.statPoints = 15;
 
-  this.equipment = {
-    armor: null,
-    weapon: null
+  this.stats = {
+    attack: 0,
+    agility: 0,
+    vitality: 0,
+    invocation: 0,
+    influence: 0,
+    shielding: 0,
+    smarts: 0,
+    willpower: 0,
+    charm: 0
```

Comment:
Whoa, that's a lot of different stats!  Let's coordinate out-of-band to go over the player experience.

Reply:
There's potential for slimming the amount down. Or at least subdividing them to make them reader-friendly (original subdivisions were `physical`, `magical`, and `mental`).